From 2ba4ecfb8113e2d22c47d401eef2a231cb1c7648 Mon Sep 17 00:00:00 2001
From: Steev Klimaszewski <steev@gentoo.org>
Date: Sun, 19 Aug 2012 05:22:58 -0500
Subject: [PATCH 2/2] Add gmodule check for libdocument

With glib 2.32 gmodule libraries have to be speficied to be linked.  This is most noticable with the gold linker.

Signed-off-by: Steev Klimaszewski <steev@gentoo.org>
---
 configure.ac            |    1 +
 libdocument/Makefile.am |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index 621e295..ab80898 100644
--- a/configure.ac
+++ b/configure.ac
@@ -169,6 +169,7 @@ PKG_CHECK_MODULES(LIBDOCUMENT, gtk+-$GTK_API_VERSION >= $GTK_REQUIRED gio-2.0 >=
 PKG_CHECK_MODULES(LIBVIEW, gtk+-$GTK_API_VERSION >= $GTK_REQUIRED gail$GAIL_API_VERSION >= $GTK_REQUIRED gthread-2.0 gio-2.0 >= $GLIB_REQUIRED)
 PKG_CHECK_MODULES(BACKEND, cairo >= $CAIRO_REQUIRED gtk+-$GTK_API_VERSION >= $GTK_REQUIRED)
 PKG_CHECK_MODULES(FRONTEND_CORE, gtk+-$GTK_API_VERSION >= $GTK_REQUIRED gthread-2.0 gio-2.0 >= $GLIB_REQUIRED)
+PKG_CHECK_MODULES(GMODULE, gmodule-2.0 >= $GLIB_REQUIRED)
 
 SHELL_PLATFORM_PKGS=
 case "$with_platform" in
diff --git a/libdocument/Makefile.am b/libdocument/Makefile.am
index cf8c555..b53fce1 100644
--- a/libdocument/Makefile.am
+++ b/libdocument/Makefile.am
@@ -114,6 +114,7 @@ libatrildocument_la_LDFLAGS = \
 
 libatrildocument_la_LIBADD = \
 	$(LIBDOCUMENT_LIBS)	\
+	$(GMODULE_LIBS)		\
 	$(top_builddir)/cut-n-paste/synctex/libsynctex.la
 
 
-- 
1.7.3.4

